### PR TITLE
Release 0.17.1

### PR DESCRIPTION
**Pull request by zimlu02**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Changelog
 All notable changes to the COBOL Language Support extension are documented in this file.
 
-## [0.17.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.16.1...0.17.0) (2021-02-02)
+## [0.17.1](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.16.0...0.17.1) (2021-02-10)
```
Comment (temanbrcom) [inline CHANGELOG.md:4]: Why do you add it here? It should be a new paragarph

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,9 @@ All notable changes to the COBOL Language Support extension are documented in th
...
 * Remove prepopulating of settings.json with empty parameters for copybooks support ([c67e75e](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/c67e75ea035c6cc9ad8904448ff3863f33944498)), closes [#701](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/701)
 * Support variable references in FILE-CONTROL ([805bf71](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/805bf7185d5fa0178a07eb1db9c5ce40d1c911d5)), closes [#718](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/718)
+* Add COPY IDMS support [#704](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/704) ([9911e95](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/9911e95d672a57f27c7f3303e68e65c1e6ab0bf2))
+* Add NPM scripts ([316e7e5](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/316e7e51721a65d5a85e4cb89119b05b7fbb6dea))
+* Initial commit for the UI regression suite ([9869f97](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/9869f976a7a67e45082154c9ff3ebc0d5e457e2f))
```
Comment (temanbrcom) [inline CHANGELOG.md:29]: Fix false-positive syntax error on SQL keywords used as variables [#766](https://github.com/eclipse/che-che4z-lsp-for-cobol/pull/766)

**Review by temanbrcom — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (zimlu02) [reply CHANGELOG.md:4]: Should we list 0.17.0 when it has been skipped and never been released?

Comment (zimlu02) [reply CHANGELOG.md:4]: let's merge it to 0.17.1.

Comment (zimlu02) [reply CHANGELOG.md:4]: sorry, my bad, we've released 0.17.0. We have to split the changelog. Thanks

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +16,7 @@ All notable changes to the COBOL Language Support extension are documented in th
 * Fix variable recognition fail on ambguity ([86b1423](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/86b1423107ec0cfc948d9fd6bf48fb24c3926b1e)), closes [#496](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/496)
 * Fix variable usages in conditions [#729](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/729) ([8f76b70](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/8f76b707c1047bc6b16fd19e4abd1d9e48af65dc))
 * Fix MAPFAIL semantic error in HANDLE CONDITION [#668](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/668)
+* Fix false-positive syntax error on SQL keywords used as variables [#766](https://github.com/eclipse/che-che4z-lsp-for-cobol/pull/766)
```
Comment (temanbrcom) [inline CHANGELOG.md:19]: This should be in the 0.17.1. The same should be in the client changelog

**Review by temanbrcom — CHANGES_REQUESTED** [review]

**Review by temanbrcom — APPROVED** [review]
🍺

**Review by grianbrcom — APPROVED** [review]
🥃

**Pull request merged**